packages/chart/src/components/BarChart.tsx

```diff
@@ -94,6 +95,36 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
     return Math.ceil(context.measureText(text).width)
   }
 
+  // This cleans a data set by:
+  // - removing commas and $ signs from any numbers to try to plot the point
+  // - removing any data points that are NOT composed of of all digits (but allow a decimal point)
+  // Without this the charts "break" and do not render
+  const cleanData = (data, testing = false) => {
```

Comment:
Although the functions differ between Pie/Bar/etc charts, I would still try lifting this up to CdcChart. You can take the visualization type as a parameter to run additional logic if needed. Something like:

`const cleanData = (data, testing = false, visType)`

In some visualizations you're logging while in others your not.

Reply:
Yes I can do that but it makes the inbound data look like:
<BarStack data={cleanData(data,false,"bar")} 
Is that what you want?